openmoc/krylov.py

```diff
@@ -8,18 +8,17 @@
 # @date June 30, 2015
 
 import sys
-import copy
+
 import numpy as np
-import scipy.sparse.linalg as linalg
```

Comment:
This makes `scipy` an optional dependency for the `openmoc` module, and only a required dependency if a user wants to use the `openmoc.krylov.IRAMSolver`.